docs/source/features/parallelisms.rst

```diff
@@ -1,56 +1,48 @@
 .. _parallelisms:
 
 Parallelisms
-------------
+============
 
-NeMo Megatron supports five types of parallelism (which can be mixed together arbitrarily).
+NeMo Megatron supports various data- and model-parallel deep learning workload deployment methods (which can be mixed together arbitrarily).
```

Comment:
NeMo Megatron supports various data- and model-parallel deep learning workload deployment methods, which can be mixed together arbitrarily.